### DISCUSSION TITLE
Download osu maps from the phone

### DISCUSSION THREAD

**Discussion by RealJevens** [ppy/osu#24952] [Ideas]
what if we can from the phone download a map to the osu direct?what I mean is, from our phone we click "download with osu direct" in a map in the web and when we log into osu in our pc, if it has internet the map automatically downloads, will be helpful because some of us scroll through twitter and/or reddit with our phones and find maps that look cool enough, but is a little tiring log into osu and search for the map the time we can play and all stuff, idk, also supporter doesn't have that much features, so probably would be like a thing to attract new users into buy it

**Comment by nagi-desuuu** [comment]
I'm afraid the next best thing would be to add them into your favorites so that it acts as some sort of "download list"
